sdk/js/src/token_bridge/getIsTransferCompleted.ts

```diff
@@ -64,6 +72,74 @@ export async function getIsTransferCompletedTerra(
   return false;
 }
 
+/**
+ * Return if the VAA has been redeemed or not
+ * @param tokenBridgeAddress The Injective token bridge contract address
+ * @param signedVAA The VAA that is being checked for redemption
+ * @param walletPKHash Wallet information
+ * @returns true if the VAA has been redeemed.
+ */
+export async function getIsTransferCompletedInjective(
+  tokenBridgeAddress: string,
+  signedVAA: Uint8Array,
+  walletPKHash: string,
+  network: any
```

Comment:
what does this argument mean? and what is its type?

Reply:
This is going away